workflow/generate_baseline_report.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env pythona
+import os
+import shutil
+import subprocess
+
+from git import Repo
+from asapp.common import config
+from workflow.evaluate_model import run as evaluate_model
```

Comment:
Any reason for doing it this way? Thoughts on putting this into a class `EvaluateModel`, importing that, then calling `.run` or `.evaluate`, kinda thing?

Reply:
Yeah I'm in the midst of changing this. I already had all this code written before we talked about using .run so was in the middle of changing them into classes that can use a base class